src/models/order.dao.js

```diff
@@ -0,0 +1,20 @@
+const { appDataSource} = require("./data-source");
+
+const createOrder = async(user_id, order_number, status_id, price, quantity ) => {
```

Comment:
```suggestion
const createOrder = async(userId, orderNumber, statusId, price, quantity ) => {
```
자바스크립트에서는 snake_case가 아닌 camelCase를 사용해주세요